src/komponenter/hent-initial-data/data-provider.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from '../../dispatch-type';
+import { AppState } from '../../reducer';
+import Innholdslaster from '../innholdslaster/innholdslaster';
+import Feilmelding from '../feilmeldinger/feilmelding';
+import { State as OppfolgingState } from '../../ducks/oppfolging';
+import { FeatureToggleState, jobbsokerbesvarelseToggleKey, servicekodeToggleKey } from '../../ducks/feature-toggles';
+import { hentServicegruppe, State as ServicegruppeState } from '../../ducks/servicegruppe';
+import { hentSykmeldtInfo, State as SykmeldtInfodataState } from '../../ducks/sykmeldt-info';
+import { hentJobbsokerbesvarelse, State as JobbsokerbesvarelseState, } from '../../ducks/jobbsokerbesvarelse';
+import getStore from '../../store';
+import { STATUS } from '../../ducks/api';
+import { ActionType } from '../../ducks/actions';
+
+interface OwnProps {
+    children: React.ReactNode;
+}
+
+interface StateProps {
+    oppfolging: OppfolgingState;
+    features: FeatureToggleState;
+    servicegruppe: ServicegruppeState;
+    sykmeldtInfo: SykmeldtInfodataState;
+    jobbsokerbesvarelse: JobbsokerbesvarelseState;
+}
+
+interface DispatchProps {
+    hentServicegruppe: () => void;
+    hentSykmeldtInfo: () => void;
+    hentJobbsokerbesvarelse: () => void;
+}
+
+type Props = StateProps & DispatchProps & OwnProps;
+
+class DataProvider extends React.Component<Props> {
+    componentWillMount() {
+        const featureJobbsokerbesvarelse = this.props.features[jobbsokerbesvarelseToggleKey];
+        const featureServicegruppe = this.props.features[servicekodeToggleKey];
+
+        this.props.hentSykmeldtInfo();
+
+        if (featureJobbsokerbesvarelse && this.props.oppfolging.data.underOppfolging) {
+            this.props.hentJobbsokerbesvarelse();
+        } else {
+            getStore().dispatch({
```

Comment:
Hva tankene rundt denne?

Reply:
Her tenkes det at vi kun henter Jobbsøkerkompetansebesvarelse hvis bruker er under oppfølging. For brukere som ikke er under oppfølging blir det sendt en dispatch med status OK slik at ikke appen stopper opp (avhengigheter innholdslaster). Ser det ikke riktig ut?

Reply:
Jeg stusser litt over at appen ikke går via react-redux på samme måte som ved resten av dispatchingen

Reply:
Fikset 👍